lib/issuer.js

```diff
@@ -188,33 +230,206 @@ api.claim = function(actor, id, callback) {
   });
 };
 
-bedrock.events.on('bedrock-mongodb.ready', function(callback) {
-  async.auto({
-    createIndexes: function(callback) {
-      // create indexes for claimed status
-      database.createIndexes([{
-        collection: store.provider.name,
-        fields: {recipient: 1, 'credential.sysState': 1, id: 1},
-        options: {unique: true, background: false}
-      }, {
-        collection: store.provider.name,
-        fields: {issuer: 1, 'credential.sysState': 1, id: 1},
-        options: {unique: true, background: false}
-      }], callback);
+// TODO: more acceptable formats?
+function checkHeaders(req, res, next) {
+  var acceptableContentTypes = [
+    'application/json',
+    'application/ld+json',
+    'text/html'
+  ];
+  if(!req.accepts(acceptableContentTypes)) {
+    res.status(406);
+    res.write(JSON.stringify({
+      message: 'Requested content types not acceptable.',
+      details: {
+        accepted: req.get('accept'),
+        acceptable: acceptableContentTypes
+      }
+    }));
+    res.end();
+    return next();
+  }
+  if(acceptableContentTypes.indexOf(req.get('Content-Type')) == -1) {
```

Comment:
Why is request 'Content-Type' (this is for the mime type of the request body) being compared to acceptable content types? Should this be using the `Accept` header instead? How is this check different (or meant to be different) from the `req.accepts` check above? Are the accept type and content type being bound together somehow here? (confusion).